pages/k8s/1.33/release-notes.md

```diff
@@ -17,6 +17,24 @@ layout:
 toc: False
 ---
 
+# 1.33+ck1
+
+### July 8, 2025 - `charmed-kubernetes --channel 1.33/stable`
+
+## Notable Fixes
+
+### What's new
```

Comment:
There feels like duplication here in these two headings. We can probably remove the What's new one